D1673/blas_interface.md

```diff
@@ -2003,8 +2006,6 @@ this.
 
   * `Real` is `float`, `double`, or `long double`.
 
-  * `x.rank()` and `y.rank()` are both one.
-
```

Comment:
Did you mean to remove the constraint on their ranks?

Reply:
Yes -- that constraint was already imposed above, as one of the requirements on template parameters with "vector" in their name.